openmdao/visualization/xdsm_viewer/xdsm_writer.py

```diff
@@ -687,13 +705,24 @@ def __init__(self, name='pyxdsm', box_stacking=_DEFAULT_BOX_STACKING,
                 Defaults to False.
             add_component_indices : bool
                 If true, display components with numbers.
-            equations : bool, optional
```

Comment:
Did the "equations" feature go away, or does the new version of pyXDSM always render indepvarcomps with equations?  This option has further integration in "om.py" as a command-line argument to `openmdao xdsm`, so it might need to be removed there (and anywhere in between).  Alternatively, maybe the option needs to be included for those still running legacy.

Reply:
I implemented it outside the class, this was by mistake left in the code and now cleaned up. In the functionality nothing changed.